README.md

```diff
@@ -1,69 +1,39 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Climb Time 
 
-## Available Scripts
+## See it Live! 
+Climb Time on Github Pages
 
-In the project directory, you can run:
+## Set-Up Front-End:
+- Clone this repo
+- Run npm install from the root directory
+- Run npm start and visit localhost:3000 in your browser
 
-### `npm start`
+## Background and Information:
+There are a few websites that are dedicated to the climbing community with varying amounts of detailed information. Typically as you try to set up your weekend excursion you are talking with a friend going through a local guidebook, with outdated trailhead information and postit notes marking the pages that you want to remember. We thought there needs to be an easier option for people to find and track the climbs here in Colorado. 
 
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+With Climb Time you can search by distance from Denver to find the style of climb you are looking to try this weekend. Once you have decided to head out for a quick Top Rope session this weekend you can set the distance willing to travel to find the areas with that style of climb. From there you will be able to see the Route name, difficulty and a link to some more beta if needed. 
```

Comment:
Some possible rewording for the first sentence: With Climb Time you can search for classic climbing routes by distance from Denver to find the style of climb you are looking to try this weekend.

You might want to consider removing "weekend" from the second sentence since you already had it in the first sentence. Can you please lowercase "Top Rope" and "Route" as well? Not sure if that was intentional or if you were trying to emphasize those words for a particular reason.